core/src/main/java/org/mini2dx/breakout/Box.java

```diff
@@ -45,15 +45,20 @@ public String toString() {
 
     void update(float delta) {
         collisionBox.preUpdate();
+        if (CollisionHandler.getCollisionHandler().getTouchedBox() == this)
+            setAlive(false);
     }
 
     void interpolate(float alpha) {
         collisionBox.interpolate(null, alpha);
     }
 
     void render(Graphics g) {
-        if (isAlive)
+        if (isAlive) {
             g.drawSprite(boxSprite, collisionBox.getRenderX(), collisionBox.getRenderY());
+            if ((BreakoutGame.DEBUG_MODE & BreakoutGame.DEBUG_COLLISION_DRAW_BOXES) != 0)
```

Comment:
If statements should always use brackets { } around the code associated with that condition - improves readability.